src/functions-and-arrays.js

```diff
@@ -1,19 +1,94 @@
 // Iteration #1: Find the maximum
 
+//console.log(Math.max(1, 3, 2));
+
+function maxOfTwoNumbers(a, b) {
+  return Math.max(a, b);
+  /*
+  if (a > b) {
+    return a;
+  }
+  else {
+    return b;
+  }
+  */
+} 
+console.log("Iteration #1: Find the maximum:", maxOfTwoNumbers(2, 4));
+
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+// let wordsLength = [7, 7, 7, 9];
+// console.log(wordsLength.indexOf(9)); // 3
+
+function findLongestWord(anyArray) {
+  lengthWords = [];
+  for (let i = 0; i < anyArray.length; i++) {
+    lengthWords.push(anyArray[i].length);
+  }
+  
```

Comment:
here you could just add
`if (words.length === 0) return null;`
to check if it is an empty array.